code/modules/antagonists/heretic/magic/mansus_grasp.dm

```diff
@@ -10,7 +10,7 @@
 	school = SCHOOL_EVOCATION
 	cooldown_time = 10 SECONDS
 
-	invocation = "R'CH T'H TR'TH!"
+	invocation = "Ftaste-tin' alithea!"
```

Comment:
Which language is this anyways, since Mansus Grasp is given to them all? 

If there's any spell that we can bend the mold on a bit (To keep `Reach the truth`) I think it'd be this one

Reply:
Oh nvm I just saw it's in greek. Still tho